pkg/gencmd/options.go

```diff
@@ -0,0 +1,19 @@
+package gencmd
+
+import (
+	"github.com/izumin5210/grapi/pkg/grapicmd"
+)
+
+type Option func(*Ctx)
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Option should have comment or be unexported